### ISSUE TITLE
Here are ideas that I have rolling around in my head

### ISSUE THREAD

**Issue by pollymonosumo93** [dawsontech30000/welcometonightmare-idea-board#1]
Maybe add environmental tech to the pack: If it would be added to the pack maybe have it start being used 65% to 75% through the pack and have alchemical coal ore gotten from void ore miner tier 6 with no modifiers at very little chance 

maybe add Divine journey to the pack: if you would add Divine journey to the pack you could have each dragon from Ice and fire mod in a separate Divine journey dim.

Maybe mess with the speed of the Item transportation system for all mod that has that so it would fit better along with progression.

**Comment by dawsontech30000** [comment]
alrighty could be possble thing to done  still want to limited project e so not all ingots rather you grind ingots etc

**Comment by pollymonosumo93** [comment]
you could have it so that only impure dust for each ingot could be gotten from the projecte and people deed to a crafting chain to change it into ingot, that is an idea to limit projecte in away but still have it usable

**Comment by dawsontech30000** [comment]
as i have said before wont be making project e right at start

**Comment by dawsontech30000** [comment]
respond when i can terms of ideas please sepreate ideas in mulitple issues so that can comment correctly

**Comment by pollymonosumo93** [comment]
Yeah and person get access to that impure dust from the table between 87.5% to 92.5% through the pack

**Comment by dawsontech30000** [comment]
confirm you talking about this?
![image](https://github.com/dawsontech30000/welcometonightmare-idea-board/assets/46389381/78533728-dbae-4030-8555-91bd49df908d)

**Comment by dawsontech30000** [comment]
if not please screenshot what talking about

**Comment by dawsontech30000** [comment]
can you do me favour tell me what mean by  "impure dust"

**Comment by pollymonosumo93** [comment]
The thing I am talking about is there would maybe be a custom item added through Contentweaker in the Projecte table maybe called impure Iron ore:

In step one person would need to convert impure Iron ore to Impure Iron dust by pulverizing it.

In step two person would need to convert Impure Iron dust to clean impure Iron dust by using water in some way maybe using fluid transposer from thermal ore something that person can imagine it is being washed

In step three would need to convert clean impure Iron dust into dirty Iron liquid by using chemical to smelt it 

In step four person would convert dirty Iron liquid to Iron Ingots and dirty liquid using power.

But this maybe gives a person either 8 Iron Ingots for one dirty Iron liquid(8x) or 16 Iron Ingots for one dirty Iron liquid(16x) or 32 Iron Ingots for one dirty Iron liquid(32x) or 64 Iron Ingots for one dirty Iron liquid(64x)

This is what I am talking about

**Comment by dawsontech30000** [comment]
this is good idea ^

**Comment by dawsontech30000** [comment]
where does project e come in this place?

**Comment by dawsontech30000** [comment]
perfer this
In step one person would need to convert impure Iron ore to Impure Iron dust by pulverizing it.

In step two person would need to convert Impure Iron dust to clean impure Iron dust by using water in some way maybe using fluid transposer from thermal ore something that person can imagine it is being washed

In step three would need to convert clean impure Iron dust into dirty Iron liquid by using chemical to smelt it

In step four person would convert dirty Iron liquid to Iron Ingots and dirty liquid using power.

But this maybe gives a person either 8 Iron Ingots for one dirty Iron liquid(8x) or 16 Iron Ingots for one dirty Iron liquid(16x) or 32 Iron Ingots for one dirty Iron liquid(32x) or 64 Iron Ingots for one dirty Iron liquid(64x)

**Comment by dawsontech30000** [comment]
but really dont want early game project  e  still

**Comment by dawsontech30000** [comment]
2-3 working days take some this ideas m

**Comment by pollymonosumo93** [comment]
> where does project e come in this place?

it gives you the impure Iron ore to start the chain

**Comment by dawsontech30000** [comment]
stick with my trees idea
sadly

**Comment by dawsontech30000** [comment]
use your idea make custom ore processing - project e part

**Comment by dawsontech30000** [comment]
iron ore-pure-Iron dust-dirty Iron liquid(8x) -16 Iron Ingots for one dirty Iron liquid(16x) or 32 Iron Ingots for one dirty Iron liquid(32x) or 64 Iron Ingots for one dirty Iron liquid(64x)

**Comment by dawsontech30000** [comment]
use your idea for custom ore proccssing seems cooler that way

**Comment by pollymonosumo93** [comment]
Like I said I am good at coming up with abstract structure for an idea to be made from anything further than that I am not good at at the momment

**Comment by dawsontech30000** [comment]
however can sepreate your ideas in to mulitple issues
please

**Comment by dawsontech30000** [comment]
just let you know that respond time going be limited today
any other new ideas please open new issue

**Comment by pollymonosumo93** [comment]
> however can sepreate your ideas in to mulitple issues please

I will do that

**Comment by pollymonosumo93** [comment]
> just let you know that respond time going be limited today any other new ideas please open new issue

I understand, and I was not hoping you would answer me right away, I can wait week or more for your answer to my messages
